packages/analytics/amplify_analytics_pinpoint/example/integration_test/events_test.dart

```diff
@@ -135,11 +97,129 @@ void main() {
               // There is some lossiness behavior to doubles
               Map.fromEntries([lossyDoubleProperty, intProperty]),
             ),
+            // Session is valid before timestamp
+            containsPair(
+                'endpoint',
+                allOf(
+                  containsPair('EndpointStatus', 'ACTIVE'),
+                  containsPair('OptOut', 'ALL'),
+                  containsPair(
+                    'Location',
+                    allOf(
+                      containsPair('Latitude', 0.0),
+                      containsPair('Longitude', 0.0),
```

Comment:
Why would these be 0s?

Reply:
That's the default value, but you're right we shouldn't need to be checking them.